packages/lodestar/src/api/rest/routes/validator/subscribeToCommitteeSubnet.ts

```diff
@@ -0,0 +1,71 @@
+import {IFastifyServer} from "../../index";
+import fastify, {DefaultBody, DefaultHeaders, DefaultParams, DefaultQuery} from "fastify";
+import {IApiModules} from "../../../interface";
+import {verify} from "@chainsafe/bls";
+import {hexToBuffer} from "../../../../util/hex";
+import {hashTreeRoot} from "@chainsafe/ssz";
+import {getDomain} from "@chainsafe/eth2.0-state-transition";
+import {computeEpochAtSlot, DomainType} from "@chainsafe/lodestar-validator/lib/util";
+
+
+const opts: fastify.RouteShorthandOptions = {
+  schema: {
+    body: {
+      type: "object",
+      requiredKeys: ["committee_index", "slot", "slot_signature", "aggregator_pubkey"],
+      "committee_index": {
+        type: "string"
+      },
+      "slot": {
+        type: "string"
+      },
+      "slot_signature": {
+        type: "string"
+      },
+      "aggregator_pubkey": {
+        type: "string"
+      },
+    },
+  }
+};
+
+interface IBody extends DefaultBody {
+  committee_index: string;
+  slot: string;
+  slot_signature: string;
+  aggregator_pubkey: string;
+}
+
+export const registerSubscribeToCommitteeSubnet = (fastify: IFastifyServer, modules: IApiModules): void => {
+  fastify.post<DefaultQuery, DefaultParams, DefaultHeaders, IBody>(
+    "/beacon_committee_subscription",
```

Comment:
question: is this up to the client to implement? Because I cant find this in https://github.com/ethereum/eth2.0-APIs/blob/master/apis/validator/beacon-node-oapi.yaml

Reply:
You are looking at wrong one. This needs to be migrated here: https://ethereum.github.io/eth2.0-APIs/

Official 0.9.x validator spec is still not merged. Here is PR version: https://mpetrunic.github.io/eth2.0-APIs/